### ISSUE TITLE
EMUwebAppConfig config editor

### ISSUE THREAD

**Issue by raphywink** [IPS-LMU/EMU-webApp#143]
In future the EMU-webApp should allow the user to configure most if not all the all the settings that are listed under the EMUwebAppConfig part of the DBconfig. The rest of the configuration of the database such as `add_levelDefinition` and so on will be handled by the emuR package as these are more invasive to the database.

**Comment by georgraess** [comment]
web frontend is progressing..... 
how should we handle communication ?
GETCONFIG and SETCONFIG inside the EMU-webApp-websocket-protocol ?
do we transfer everything (the whole json obj) at once ? or single values ?

**Comment by raphywink** [comment]
I don't think we need an explicit "getter" command as the EMU-webApp already has the EMUwebAppConfig stored in the ConfigProviderService after the GETGLOBALDBCONFIG protocol call is made. What we do need is something like a SAVEEMUWEBAPPCONFIG command in the protocol. This then would send the entire EMUwebAppConfig section of the _DBconfig file to the serve. The serve then saves the changes and maybe the current bundle is reloaded to apply the changes made to the EMUwebAppConfig

**Comment by georgraess** [comment]
communication with server still has to be done
